onnx/compose.py

```diff
@@ -421,6 +421,7 @@ def add_prefix_graph(
     rename_initializers: Optional[bool] = True,
```

Comment:
nit: Do these flags ever take None? If not I would type them as `bool` instead of `Optional[bool]`